src/client/src/pages/settings/adminSettings.js

```diff
@@ -1326,33 +1334,24 @@ class AdminSettings extends React.Component {
 
 AdminSettings.propTypes = {
   developer: PropTypes.object,
-  listUsers: PropTypes.func,
   listWorkgroups: PropTypes.func,
   reloadUser: PropTypes.func,
   t: PropTypes.func,
   user: PropTypes.object,
   users: PropTypes.object,
 };
 
-// AdminSettings.defaultProps = {
-// };
-
 const mapStateToProps = state => {
   return {
     developer: state.developer,
     user: state.core_user,
-    users: state.core_users,
     workgroups: state.core_workgroups,
   };
 };
 
 const mapDispatchToProps = dispatch => {
   return {
     dispatch: dispatch,
```

Comment:
Q: Die User werden jetzt gar nicht mehr in den redux store geschrieben. richtig? Hat das noch Auswirkungen auf andere Teile der Applikation?

Reply:
Ja, das ist richtig. Die `core_users` wurden nur in der Admin-View verwendet und deshalb habe ich sie aus dem Store entfernt.